### PR TITLE
Feature/product page

### PR DESCRIPTION
**Pull request by snafx**

### CODE REVIEW

**Review by lukaszwasielewski — COMMENTED** [review]
great job!

**Comment by UlaR11** [comment]
well done!

File: src/main/java/com/sda/Warehouse/controllers/ProductController.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class ProductController {
+
+    @Autowired
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/controllers/ProductController.java]: Filed injection is considered as a bad practise, use contructor injection :)

File: src/main/resources/templates/product.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        <h1>Sda warehouse</h1>
+        <nav class="navbar navbar-inverse sticky-top">
+            <div class="container-fluid">
+                <div class="navbar-header">
+                    <a class="navbar-brand" href="#">WebSiteLogo</a>
+                </div>
+                <ul class="nav navbar-nav">
+                    <li class="active"><a href="#">Home</a></li>
+                    <li><a href="#">Search</a></li>
+
```
Comment (RafalCiolek) [inline src/main/resources/templates/product.html]: useless line

File: src/main/resources/templates/product.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                    convallis
+                    nunc.
+                    Fusce neque neque, fermentum in molestie vitae, tincidunt nec nulla. Donec eleifend risus vel urna
+                    facilisis,
+                    sit amet euismod ipsum rutrum.</p>
+            </div>
+        </div>
+    </div>
+</div>
+
```
Comment (RafalCiolek) [inline src/main/resources/templates/product.html]: useless line

**Comment by snafx** [comment]
code refactor, please say if it's ok now

**Review by RafalCiolek — APPROVED** [review]

**Pull request merged**